src/Layout/__tests__/GridContainer.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { strictEqual } from 'assert';
+import { getGridColumnStyles } from '../GridContainer';
+
+describe('getGridColumnStyles function', function () {
+  it('should return the correct grid-column value for placement "left"', function () {
```

Comment:
If my aim is to make sure that the "left" placement is working properly, rather than specifically "left" with width 12, would it instead make sense to write it like this?
```suggestion
  it('returns correct grid-column for placement "left"', function () {
    strictEqual(getGridColumnStyles('left', 5), 'span 5');
    strictEqual(getGridColumnStyles('left', 7), 'span 7');
    strictEqual(getGridColumnStyles('left', 12), 'span 12');
  });
```

Reply:
Tests were updated in commit 88104dcbff8db70e6b85aca123a517843654198d